shotover-proxy/src/codec/cassandra.rs

```diff
@@ -187,9 +187,8 @@ fn reject_protocol_version(version: u8) -> CodecReadError {
             version: Version::V4,
             stream_id: 0,
             operation: CassandraOperation::Error(ErrorBody {
-                error_code: 0xA, // https://github.com/apache/cassandra/blob/adf2f4c83a2766ef8ebd20b35b49df50957bdf5e/doc/native_protocol_v4.spec#L1053
                 message: "Invalid or unsupported protocol version".into(),
-                additional_info: AdditionalErrorInfo::Server,
+                ty: ErrorType::Protocol,
```

Comment:
Slightly surprised about the `ty` naming. I know `type` is reserved, but `error_type`? Too late lol.

Reply:
`ty` is pretty standard due to `type` being reserved, but you make a good point, the redundant `error_` probably would have been worth it to avoid abbreviating `type`